exonum/src/storage/proof_map_index/mod.rs

```diff
@@ -583,11 +586,12 @@ where
     fn insert_branch(
         &mut self,
         parent: &BranchNode,
-        key_slice: &DBKey,
+        key_slice: &ProofPath,
```

Comment:
Shouldn't this be renamed to something more "path"-related? It's not "key" any more.